app/cafe.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
```

Comment:
import groups should be seprarated by blank line
1. Default python imports
2. Installed imports
3. Your custom imports